lib/parser.rb

```diff
@@ -0,0 +1,32 @@
+# Understands how to parse natural language positions into formal ones
+
+class Parser
+  ROW_POSITION = 0
+  COL_POSITION = 1
+
+  def initialize
+    @grid_index = { top: 0, mid: 1, low: 2, one: 0, two: 1, three: 2 }
+  end
+
+  def map_choice_to_grid(choice)
```

Comment:
Should this class have knowledge of the structure of the board? Or should it be Board that has that knowledge?